webpack.common.js

```diff
@@ -10,9 +10,15 @@ module.exports = {
         ]
     },
     plugins: [
-        new CopyPlugin([{
-            from: "**/*",
-            to: "."
-        }])
+        new CopyPlugin([
+            {
+                from: "**/*",
+                to: "."
+            },
+            {
+                context: path.resolve(__dirname),
+                from: "node_modules/libass-wasm/dist/subtitles-octopus-worker.*",
```

Comment:
We recently added a new directory to replace the node_modules directory in the built client, can you change this path to match that?

Reply:
fixed